webapp/components/interop-dashboard.js

```diff
@@ -456,6 +466,28 @@ class InteropDashboard extends PolymerElement {
       },
       dataManager: Object,
       scores: Object,
+      tableState: {
+        type: Object,
+        value: {
+          sortColumn: -1,
+          sortOrder: 'asc',
+          sorted: true
+        }
+      },
```

Comment:
`tableState` can be broken into separate variables here, and the `sortOrder` property can be changed to a boolean if renamed to something like `isSortedAscending`, which should make for easier comparisons. I'm also not seeing any uses of the `sorted` property in the code (but I may have missed something).